consensus/clique/src/test/java/tech/pegasys/pantheon/consensus/clique/blockcreation/CliqueBlockCreatorTest.java

```diff
@@ -108,12 +112,17 @@ public void proposerAddressCanBeExtractFromAConstructedBlock() {
     final CliqueExtraData extraData =
         new CliqueExtraData(BytesValue.wrap(new byte[32]), null, validatorList);
 
+    final Vertx vertx = Vertx.vertx();
+    final TimerUtil timerUtil = new VertxTimerUtil(vertx);
+    final long TRANSACTION_EVICTION_INTERVAL_MS = TimeUnit.HOURS.toMillis(1);
```

Comment:
We shouldn't be introducing a real Vertx here.  just a mock or stub `TimerUtil` would be enough.
Probably should make `Transaction_EVICTION_INTERVAL_MS` an actual constant as well instead of duplicating it for each test.

Same thing for all the other test code.